scripts/debug_tools/prepare_analyzer_cmd.py

```diff
@@ -64,7 +64,24 @@ def __init__(
 def prepare(analyzer_command_file, pathOptions):
     res = lib.change_paths(get_first_line_of_file(analyzer_command_file),
                            AnalyzerCommandPathModifier(pathOptions))
-    return res
+
+
+    if '-nobuiltininc' not in res:
+        return res
+
+    # Find Clang include path
+    clang_lib_path = os.path.dirname(pathOptions.clang) + '/../lib'
+    clang_include_path = ''
+    for path, _, files in os.walk(clang_lib_path):
```

Comment:
Would `os.walk` go down in the directory tree into lower levels? E.g. in my case the file is here: `../lib/clang/5.0.0/include/stddef.h` ?

Reply:
Yes, that's the goal of `os.walk()`. It visits all subdirectories recursively.